atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -91,12 +91,12 @@
     public CassandraService(
             MetricsManager metricsManager,
             CassandraKeyValueServiceConfig config,
-            Blacklist blacklist,
+            Denylist denylist,
```

Comment:
Also a break, but this is only used internally to the best of my knowledge.

Reply:
Yeah, I am not sure how to guarantee its not used outside. Would defer to @jeremyk-91 / @gmaretic